### DISCUSSION TITLE
How to set everything path?

### DISCUSSION THREAD

**Discussion by futag** [Flow-Launcher/Flow.Launcher#2165] [Q&A] [answered]
![image](https://github.com/Flow-Launcher/Flow.Launcher/assets/21020026/e0677698-5f37-49b0-80d7-bbd0a5c48da7)

**Comment by JohnTheGr8** [comment, marked as answer]
Hi. You need to switch from `Windows Index` to `Everything` in the `General Setting` tab to enable searching with Everything.

If you are running the Everything service, you can leave `Everything Path` empty and it should work.

**Comment by newnol** [comment]
Move to the tab **General Setting" changed all the window Index into Everything.
